osu.Game/Screens/Select/FooterV2/FooterButtonFreeModsV2.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+
+namespace osu.Game.Screens.Select.FooterV2
+{
+    public partial class FooterButtonFreeModsV2 : FooterButtonV2
```

Comment:
This isn't displayed anywhere?

Reply:
im aware it isn't, thought it wasn't worth adding to the test scene since it has no special functionality and wasnt present on the test this test is replacing, and its usage case is clear, but ill add it now

Reply:
We don't add classes to the project unless they have a usage.